### ISSUE TITLE
Starting from 5.1.4 npm release, assToVtt generates incorrect VTT format

### ISSUE THREAD

**Issue by HFO4** [zhw2590582/ArtPlayer#763]
### Expected behaviour

`assToVtt` should generates correct format like below
```
WEBVTT

1
00:00:00.000 --> 00:00:05.000
This is a test of the ASS format and some basic features in it.

2
00:00:05.000 --> 00:00:07.000
This and the previous line should both show at the bottom of the video in 20 pixel high white Arial with a 2 pixel black outline and 2 pixel offset shadow. This line is long so it should automatically be broken into several lines that are approximately even length, with upper lines being longer.
```

### Actual behaviour

The generated VTT missing End of line `\n\n`:

```
WEBVTT 1 00:00:00.000 --> 00:00:05.000 This is a test of the ASS format and some basic features in it.

2 00:00:05.000 --> 00:00:07.000 This and the previous line should both show at the bottom of the video in 20 pixel high white Arial with a 2 pixel black outline and 2 pixel offset shadow. This line is long so it should automatically be broken into several lines that are approximately even length, with upper lines being longer.
```

The borowser cannot parse such VTT format, so not subtitle shown for ASS files anymore.

### Steps to reproduce

You can inspect `dist` from 5.1.4, the line break `\n\n` is somehow missing in `assToVtt`, but it did appears in `srtToVtt`:
<img width="798" alt="image" src="https://github.com/zhw2590582/ArtPlayer/assets/16058869/0e62aa1f-3c36-4952-920d-aee286da7e3b">

### Environment

-   Browser: 
-   Version: 5.1.4
-   Operating System:
-   Version:

### Console errors (if any)
None

### Link to where the bug is happening
None

**Comment by zhw2590582** [comment]
Fixed in `artplayer@5.1.5`

**Comment by HFO4** [comment]
Comfirm it's fixed. Thanks!
